common/src/main/java/whocraft/tardis_refined/common/block/StairBlock.java

```diff
@@ -0,0 +1,9 @@
+package whocraft.tardis_refined.common.block;
+
+import net.minecraft.world.level.block.state.BlockState;
+
+public class StairBlock extends net.minecraft.world.level.block.StairBlock {
```

Comment:
Perhaps it would be good to rename our versions to something more recognisable so we don't accidentally call the vanilla one and get confused.